doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -556,87 +546,72 @@ the conversion in hypotheses :n:`{+ @ident}`.
-   The tactic :tacn:`cbn` reduces whenever it will be able to reuse it or not:
-   :g:`succ t` is reduced to :g:`S t`.
-
-.. tacv:: cbn [ {+ @qualid} ]
-          cbn - [ {+ @qualid} ]
+   The behavior can be tuned using the :cmd:`Arguments` command.
 
-   These are respectively synonyms of :n:`cbn beta delta [ {+ @qualid} ] iota zeta`
-   and :n:`cbn beta delta - [ {+ @qualid} ] iota zeta` (see :tacn:`cbn`).
+.. tacn:: cbn {? @strategy_flag } {? @occurrences }
 
-.. tacv:: simpl @pattern
+   :tacn:`cbn` tactic was intended to be a more principled, faster and more
+   predictable replacement for :tacn:`simpl`.
 
-   This applies :tacn:`simpl` only to the subterms matching
-   :n:`@pattern` in the current goal.
+   :tacn:`cbn` tactic accepts the same flags as :tacn:`cbv` and
+   :tacn:`lazy`. The behavior can be tuned using the :cmd:`Arguments` command.
 
-.. tacv:: simpl @pattern at {+ @natural}
+   .. todo add "See <subsection about controlling the behavior of reduction strategies>"
+      to TBA section
 
-   This applies :tacn:`simpl` only to the :n:`{+ @natural}` occurrences of the subterms
-   matching :n:`@pattern` in the current goal.
+   The main difference between :tacn:`cbn` and :tacn:`simpl` is that
+   :tacn:`cbn` reduces whether it will be able to reuse the constant or not:
+   :g:`succ t` is reduced to :g:`S t`.
 
    .. exn:: Too few occurrences.
       :undocumented:
 
-.. tacv:: simpl @qualid
-          simpl @string
-
-   This applies :tacn:`simpl` only to the applicative subterms whose head occurrence
-   is the unfoldable constant :n:`@qualid` (the constant can be referred to by
-   its notation using :n:`@string` if such a notation exists).
-
-.. tacv:: simpl @qualid at {+ @natural}
-          simpl @string at {+ @natural}
-
-   This applies :tacn:`simpl` only to the :n:`{+ @natural}` applicative subterms whose
-   head occurrence is :n:`@qualid` (or :n:`@string`).
-
 .. flag:: Debug RAKAM
 
    This flag makes :tacn:`cbn` print various debugging information.
    ``RAKAM`` is the Refolding Algebraic Krivine Abstract Machine.
 
-.. tacn:: unfold @qualid
-   :name: unfold
+.. tacn:: unfold {+, @reference_occs } {? @occurrences }
 
-   This tactic applies to any goal. The argument qualid must denote a
-   defined transparent constant or local definition (see
-   :ref:`gallina-definitions` and
-   :ref:`vernac-controlling-the-reduction-strategies`). The tactic
-   :tacn:`unfold` applies the :math:`\delta` rule to each occurrence
-   of the constant to which :n:`@qualid` refers in the current goal
-   and then replaces it with its :math:`\beta\iota\zeta`-normal form.
-   Use the general reduction tactics if you want to avoid this final
-   reduction, for instance :n:`cbv delta [@qualid]`.
+   .. todo PR: some of the variants said they would do beta iota conversion
+      rather than beta iota zeta conversion.  I think it's probably the same
+      in either case, not sure which is right.
```

Comment:
Indeed, forgetting the `zeta` was probably an oversight. I don't see why it wouldn't be included, like in most other reduction tactics.